python/cudf/cudf/core/scalar.py

```diff
@@ -353,7 +354,12 @@ def _binaryop(self, other, op: str):
     def _dispatch_scalar_binop(self, other, op):
         if isinstance(other, Scalar):
             other = other.value
-        return getattr(self.value, op)(other)
+        try:
+            func = getattr(operator, op)
+            return func(self.value, other)
```

Comment:
Yup, that's right.

Reply:
Nice trick to fetch the operation for the forward direction only!

As a purist: it might be nice to move the function call out of the `try` clause (it could raise an `AttributeError`).

Reply:
> As a purist: it might be nice to move the function call out of the `try` clause (it could raise an `AttributeError`).

Done 👍